.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+# See https://pre-commit.com/ for usage and config
+repos:
+- repo: local
+  hooks:
+  - id: black
+    name: black
+    stages: [commit]
+    language: system
+    entry: black
+    types: [python]
+
+  # - id: isort
+  #   name: isort
+  #   stages: [commit]
+  #   language: system
+  #   entry: isort
+  #   types: [python]
+
+  # - id: flake8
+  #   name: flake8
+  #   stages: [commit]
+  #   language: system
+  #   entry: flake8
+  #   types: [python]
+
+  # - id: mypy
+  #   name: mypy
+  #   stages: [commit]
+  #   language: system
+  #   entry: mypy
+  #   types: [python]
+  #   require_serial: true
+
+  # - id: pytest-cov
+  #   name: pytest
+  #   stages: [commit]
+  #   language: system
+  #   entry: pytest --cov --cov-fail-under=90
+  #   types: [python]
+  #   pass_filenames: false
```

Comment:
could we avoid commented code and add this if/when we use these tools in pre-commit?

```suggestion
```